doc/orchestrator.rst

```diff
@@ -1,60 +1,324 @@
+``Orchestrator`` via the :ref:`SmartRedis<smartredis-api>`
+client library to transmit data, execute ML models, and execute scripts.
+
+SmartSim offers two types of orchestrator deployments: :ref:`clustered deployment<Clustered Deployment>` and
+:ref:`colocated deployment<Colocated Deployment>`. Continue to the respective deployment sections for
+information on deployment types.
+
+Below is a diagram demonstrating an orchestrator as a general feature store
+capable of storing numerical data (tensors and datasets), AI models, and scripts (TorchScript).
+Combined with the SmartRedis clients, the ``Orchestrator`` is capable of hosting and executing
+AI models written in Python on CPU or GPU. The ``Orchestrator`` supports models written with
+TensorFlow, Pytorch, TensorFlow-Lite, or models saved in an ONNX format (e.g. sci-kit learn).
 
 .. |orchestrator| image:: images/Orchestrator.png
   :width: 700
   :alt: Alternative text
 
 |orchestrator|
 
-Combined with the SmartRedis clients, the ``Orchestrator`` is capable of hosting and executing
-AI models written in Python on CPU or GPU. The ``Orchestrator`` supports models written with
-TensorFlow, Pytorch, TensorFlow-Lite, or models saved in an ONNX format (e.g. sci-kit learn).
-
-
-Cluster Orchestrator
-====================
-
-The ``Orchestrator`` supports single node and distributed memory settings. This means
-that a single compute host can be used for the database or multiple by specifying
-``db_nodes`` to be greater than 1.
+======================
+Clustered Deployment
+======================
+--------
+Overview
+--------
+During clustered deployment, a SmartSim ``Orchestrator`` (the database) runs on separate
+compute node(s) from the model node(s). A clustered orchestrator can be deployed on a single
+node or sharded (distributed) over multiple nodes.
+With a sharded orchestrator, available hardware for inference and script
+evaluation increases and overall memory for data storage increases.
+
+Communication between a clustered Orchestrator and Model
+is initialized in the application script via a SmartRedis client.
+Users do not need to know how the data is stored in a clustered configuration and
+can address the cluster with the SmartRedis clients like a single block of memory
+using simple put/get semantics in SmartRedis. The client establishes a
+connection using the database address and travels off the model compute node to
+the database compute node.
 
 .. |cluster-orc| image:: images/clustered-orc-diagram.png
   :width: 700
   :alt: Alternative text
 
 |cluster-orc|
 
+A clustered database is optimal for high data throughput scenarios
+such as online analysis, training and processing.
+Clustered Orchestrators support data communication across multiple simulations.
+With clustered database deployment, SmartSim can run AI models, and Torchscript
+code on the CPU(s) or GPU(s) with existing data in the ``Orchestrator``.
+Produced data can then requested by another application.
```

Comment:
slightly awkward wording of `Produced data can then requested by another application` might be better as "Data produced by by these processes is available for consumption by other applications" -> at min, maybe `produced data can then _be_ requested...`